packages/react-server-cli/src/compileClient.js

```diff
@@ -135,10 +135,15 @@ const packageCodeForBrowser = (entrypoints, outputDir, outputUrl, hot, minify, l
 					exclude: /node_modules/,
 				},
 				{
-					test: /.css$/,
+					test: /\.css$/,
 					loader: extractTextLoader,
 					exclude: /node_modules/,
 				},
+				{
+					test: /\.(eot|woff|woff2|ttf|ttc|png|svg|jpg|jpeg|gif|cgm|tiff|webp|bmp|ico)$/i,
```

Comment:
If folks want to, it's fine by me. Could also consider other media files (i.e. video, audio, and 3D/VR).

However, remember that with this PR that `require(foo.txt)` will still throw an error server-side, because the only `require.extension` we set is ".css", so adding txt or csv here will only help for txt and csv files _that are referenced by CSS_. That seems pretty unusual to me, and probably not worth it?